src/persist/src/indexed/encoding.rs

```diff
@@ -218,6 +223,14 @@ impl BlobTraceUpdates {
         }
     }
 
+    /// Return the estimated memory usage of the raw data.
```

Comment:
This may affect the inlining behaviour, since this is used to check whether a batch is over the inlining threshold or not. I don't see much difference in my staging env, though.

Reply:
Overall though I'm glad to make this more accurate!